mne/decoding/tests/test_receptive_field.py

```diff
@@ -404,24 +416,28 @@ def test_receptive_field_nd():
          [0, -1, 0, 0, 0, 0],
          [0, 0, 0, 0, 0, 0]],
     ]
-    tdr = TimeDelayingRidge(slim[0], slim[1], 1., 0.1, 'laplacian')
+    tdr = TimeDelayingRidge(slim[0], slim[1], 1., 0.1, 'laplacian',
```

Comment:
I think the repeated `slim[0], slim[1]` looks awkward, why not

```
slim = start, end = [0, 5]

...

tdr = TimeDelayingRidge(start, end, ...)
```

or something like that.